src/modules/deployment/utils.ts

```diff
@@ -162,7 +163,7 @@ export const getEmptyDeployment = (projectId: string): [Project, Scene] => {
         name: 'Bermuda Grass',
         model: 'FloorBaseGrass_01/FloorBaseGrass_01.glb',
         script: null,
-        thumbnail: 'https://builder-api.decentraland.org/v1/storage/assets/QmexuPHcbEtQCR11dPXxKZmRjGuY4iTooPJYfST7hW71DE',
+        thumbnail: getContentsStorageUrl('QmexuPHcbEtQCR11dPXxKZmRjGuY4iTooPJYfST7hW71DE'),
```

Comment:
I have no idea what I'm talking about but this seems to be repeated on project/sagas? The asset I mean. Would it make sense to keep it only in one place?

Reply:
done 👍